src/main/java/com/qunite/api/web/dto/queue/QueueCreationDto.java

```diff
@@ -5,5 +5,4 @@
 @Data
 public class QueueCreationDto {
   private String name;
-  private Long creatorId;
-}
+}
```

Comment:
Revert

Reply:
it`s better to get user implicitly from credentials than allow anyone create queues for other people(or add unnecessary validation)